# message for publishing Vector 2 (float 16)
float32 x
float32 y
float32 z


